src/dataworker/Dataworker.ts

```diff
@@ -178,19 +209,24 @@ export class Dataworker {
       allValidFills,
       blockRangesForChains,
       this.chainIdListForBundleEvaluationBlockNumbers
-    );
+    ).filter(sdkUtils.isV2Fill);
```

Comment:
I checked this locally and tsc isn't narrowing the type for me, so it thinks allValidFillsInRange is FillWithBlock[].

```suggestion
    ).filter(sdkUtils.isV2Fill<V2FillWithBlock, V3FillWithBlock>);
```

Reply:
Has been resolved since I made the comment.